src/generators.js

```diff
@@ -2,7 +2,7 @@ import AdminOnRestGenerator from "./generators/AdminOnRestGenerator";
 import NextGenerator from "./generators/NextGenerator";
 import ReactGenerator from "./generators/ReactGenerator";
 import ReactNativeGenerator from "./generators/ReactNativeGenerator";
-import TypescriptInterfaceGenerator from "./generators/TypescriptInterfaceGenerator";
```

Comment:
Interfaces generator is useful in certain cases, possible to keep it?

Reply:
Yes, of course. I'll make changes soon

Reply:
Check it out :+1: